cim/templates/wo-details.html

```diff
@@ -179,14 +179,20 @@ <h5 class="modal-title">Edit Details for Product Serial Number <span id="edit_Wo
 					</div>  
 
 					<div class="form-group p-1">
-						<label class="control-label" >Product Part Number:</label>
-						<select id="product_pn_to_edit" name="product_pn_to_edit">
-								<option value="Basic-i3">Basic i3</option>
-								<option value="Basic-i5">Basic i5</option>
-								<option value="Basic-i7">Basic i7</option>
-								<option value="Pro-i3">Pro i3</option>
-								<option value="Pro-i5">Pro i5</option>
-								<option value="Pro-i7">Pro i7</option>
+						<label class="control-label" >Part Number:</label>
+						<select id="product_pn_to_edit">
+								<!-- <option value="Basic_i3">Basic_i3</option>
+								<option value="Basic_i5">Basic_i5</option>
+								<option value="Basic_i7">Basic_i7</option>
+								<option value="Pro_i3">Pro_i3</option>
+								<option value="Pro_i5">Pro_i5</option>
+								<option value="Pro_i7">Pro_i7</option> -->
+								<option >Basic_i3</option>
+								<option >Basic_i5</option>
+								<option >Basic_i7</option>
+								<option >Pro_i3</option>
+								<option >Pro_i5</option>
+								<option >Pro_i7</option>
```

Comment:
this will break the js code

I used product_pn: this.product_pn_to_edit.value in the JS